test/evergreen/configure_combinations.sh

```diff
@@ -7,15 +7,17 @@ sh build_posix/reconf
 curdir=`pwd`
 
 flags="CFLAGS=\"-Werror -Wall -Wextra -Waddress -Waggregate-return -Wbad-function-cast -Wcast-align -Wdeclaration-after-statement -Wformat-security -Wformat-nonliteral -Wformat=2 -Wmissing-declarations -Wmissing-field-initializers -Wmissing-prototypes -Wnested-externs -Wno-unused-parameter -Wpointer-arith -Wredundant-decls -Wshadow -Wundef -Wunused -Wwrite-strings -O -fno-strict-aliasing -Wuninitialized\"
-CC=clang CFLAGS=\"-Wall -Werror -Qunused-arguments -Wno-self-assign -Wno-parentheses-equality -Wno-array-bounds\""
+CC=clang CFLAGS=\"-Wall -Werror -Qunused-arguments -Wno-self-assign -Wno-parentheses-equality -Wno-array-bounds\"
+CFLAGS=\" \""
```

Comment:
What's the purpose of this line?

Reply:
This is to test a scenario where no explicit value is set for `CFLAGS`.

Reply:
In that case, I'd rather leave without this line as we've got some coverage for that (e.g. `ubuntu1804` build variant). Also, adding one line here actually means adding tests for a list of configure options (listed below), which could extend the run time of this test by 1/2.